app/uk/gov/hmrc/slacknotifications/services/NotificationService.scala

```diff
@@ -138,9 +138,11 @@ class NotificationService @Inject()(
       }
       .flatMap {
         case Some(slackChannel) => f(slackChannel)
-        case None               => Future.successful(NotificationResult().addError(SlackChannelNotFoundForTeamInUMP(teamName)))
+        case None               => f(predictedTeamName(teamName))
       }
 
+  private def predictedTeamName(teamName: String): String = "team-" + teamName.replace(" ", "-").toLowerCase
```

Comment:
how does it fail if the prediction doesn't exist? Does it report that the team channel doesn't exist?

Reply:
Yeah, you'll get a `SlackChannelNotFound` error instead.